src/lifetracker/logic/LogicImpl.java

```diff
@@ -42,7 +46,23 @@ public ExecuteResult executeCommand(String commandString) {
         if (commandString.equals("exit")) {
             return runResult;
 
-        } else {
+        }
+        
+        else if (commandContent[0].equals("saveat")) {
```

Comment:
Should the UI show an empty list and a comment like "The calendar is saved at......"?